packages/salesforcedx-utils-vscode/test/unit/context/workspaceContextUtil.test.ts

```diff
@@ -120,14 +68,14 @@ describe('WorkspaceContext', () => {
 
   let getUsernameStub: SinonStub;
   let getUsernameOrAliasStub: SinonStub;
-  let workspaceContextUtil: typeof WorkspaceContextUtil;
+  let workspaceContextUtil: any; // TODO find a better way
```

Comment:
Probably create a MockExtensionContext class, then on line 81, have it be:
```
const mockExtensionContext = new MockExtensionContext;
```

...and on line 96:
```
await workspaceContextUtil.initialize(mockExtensionContext);
```

Something to do for later.